official/keras/cifar10_dataset.py

```diff
@@ -0,0 +1,32 @@
+import keras
+from keras.datasets import cifar10
+import numpy as np
+import tensorflow as tf
+
+def get_train_eval_dataset():
```

Comment:
It would be preferable to use something from core TF for CIFAR10. Do we have it in TF?

Reply:
Yes, I agree. This was one of the things I was planning on refactoring. I found an example in the model repo that converts the cifar10 dataset to TFRecordDataset. We probably need to write a new one.